### ISSUE TITLE
P3: When adding a filter/aggregate/groupby: keyboard focus should be on first select

### ISSUE THREAD

**Issue by yaelleC** [grafana/azure-data-explorer-datasource#486]
When using the keyboard the create an ADX query, and hitting the `+` button to create a filter/agregate/group by item, the focus is lost. 
It would be nice to have focus on the first element of the newly created form instead.

We have set up this issue as lowest a11y prio as it requires quite a lot of effort 

https://user-images.githubusercontent.com/42030685/194829786-988eeb6f-472d-419f-b9b5-48881fee9c41.mov

To Do

- [x] Get UX input to confirm the wanted behaviour for the form
- [ ] Implement the changes

**Comment by andresmgot** [comment]
does this happens with the new editor? (this editor will go away)

**Comment by asimpson** [comment]
@andresmgot Yep, it also happens on the new Editor. 

@diegoadams or @catherineymgui any thoughts from a UX perspective here?

**Comment by catherineymgui** [comment]
Yeah, I agree that there should be focus on the first element once a new filter is created.
